### PR TITLE
tikv-server: Tune rocksdb options

### PR DESCRIPTION
**Pull request by shenli**
Set opts.set_max_bytes_for_level_base to 64MB.
Make data importing process stable.

@ngaut

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -97,9 +97,12 @@ fn build_raftkv(matches: &Matches,
     opts.set_max_write_buffer_number(5);
     opts.set_min_write_buffer_number_to_merge(2);
     opts.set_max_background_compactions(3);
-    opts.set_max_bytes_for_level_base(512 * 1024 * 1024);
+    opts.set_max_bytes_for_level_base(64 * 1024 * 1024);
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: Why 64M? The level base should be approximate `write_buffer_size * min_write_buffer_number_to_merge * level0_file_num_compaction_trigger`, which is 512M.

Comment (c4pt0r) [inline src/bin/tikv-server.rs]: According to usual practice, the size of L0 should be approximately equal to the size of L1, the max_bytes_for_level_base should be <the size of L0 file> * options.level0_file_num_compaction_trigger. In our case, L0 is ~10M, so make max_bytes_for_level_base 64M is OK.

Comment (shenli) [inline src/bin/tikv-server.rs]: The size of level 0 is about 10MB and the size of level 1 should be similar to the size of level 0. So max_bytes_for_level_base should be Size_of_Level0_file * level0_file_num_compaction_trigger. The default value of level0_file_num_compaction_trigger is 4. So I set max_bytes_for_level_base to 60MB.  512Mb is too big.

Refer:
1. Siying Dong's suggestion in email. 
2. https://github.com/facebook/rocksdb/wiki/RocksDB-Tuning-Guide
"We usually reommend making L0->L1 as fast as possible by making the size of level 0 similar to size of level 1."

**Comment by c4pt0r** [comment]
LGTM

File: src/bin/tikv-server.rs (hunk 1 above)
Comment (BusyJay) [inline src/bin/tikv-server.rs]: I guess that's because we turn on compression in practice. This option may not be changed without compression turned on.

Comment (shenli) [inline src/bin/tikv-server.rs]: Yes. So I add a TODO and call for you guys install lz4 lib. We will enable compression soon.

File: src/bin/tikv-server.rs [hunk 2]
```diff
@@ -97,9 +97,12 @@ fn build_raftkv(matches: &Matches,
...
     opts.set_target_file_size_base(64 * 1024 * 1024);
     opts.create_if_missing(true);
+    opts.set_level_zero_slowdown_writes_trigger(12);
+    opts.set_level_zero_stop_writes_trigger(24);
+    opts.compression(DBCompressionType::DBLz4);
```
Comment (shenli) [inline src/bin/tikv-server.rs]: @BusyJay compression enabled.

**Comment by ngaut** [comment]
LGTM

File: src/bin/tikv-server.rs (hunk 2 above)
Comment (BusyJay) [inline src/bin/tikv-server.rs]: Why not just edit L95?

**Comment by shenli** [comment]
PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**